tests/baselines/reference/user/async.log

```diff
@@ -51,7 +51,7 @@ node_modules/async/autoInject.js(160,28): error TS2695: Left side of comma opera
 node_modules/async/autoInject.js(164,14): error TS2695: Left side of comma operator is unused and has no side effects.
 node_modules/async/autoInject.js(168,6): error TS2695: Left side of comma operator is unused and has no side effects.
 node_modules/async/cargo.js(62,12): error TS2304: Cannot find name 'AsyncFunction'.
-node_modules/async/cargo.js(67,14): error TS2591: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i @types/node` and then add `node` to the types field in your tsconfig.
+node_modules/async/cargo.js(67,14): error TS2749: 'module' refers to a value, but is being used as a type here.
```

Comment:
Seems like a regression from #29896, although I think maybe both errors should not happen in JS. Need to investigate.

Reply:
Yeah, this is just horrible misunderstanding of the jsdoc `module:QualifiedName` syntax.